### PR TITLE
clang check modernize-use-nullptr

### PR DESCRIPTION
**Pull request by davidlange6**
For comment/testing purposes (certainly will become unmergable soon, but not yet)

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

**Comment by davidlange6** [comment]
abort

**Comment by davidlange6** [comment]
please test

File: HLTrigger/HLTcore/src/HLTConfigData.cc [hunk 1]
```diff
@@ -18,7 +18,7 @@
 static const edm::ParameterSet initializeDummyPSet() {
   edm::ParameterSet dummy;
   dummy.registerIt();
-  return std::move(dummy);
+  return dummy;
```
Comment (fwyzard) [inline HLTrigger/HLTcore/src/HLTConfigData.cc]: did this get changed by `modernize-use-nullptr` ?

**Comment by davidlange6** [comment]
I think not - there appears to be some "default" checks to remove if you want to run one tool at a time.  I've started over with a less buggy script

**Comment by slava77** [comment]
@davidlange6 
is there a visible cost to rebase or similar actions to step over a commit that touches ~all of CMSSW?

**Comment by davidlange6** [comment]
presumably one would not follow the approach of 1 PR per tool when making PRs for committing. One pr per subsystem for all tools could be better. But first we discussed enforcing on files touched by incoming PRs (which comes after knowing what to do)

**Pull request closed without merging**